Build error: unknown type name 'FOUNDATION_EXPORT'

I was consuming libPhoneNumber-iOS using the Swift Package Manager, but after updating the package to version 1.3.0 my build started failing. I'm using the library in an iOS application that I'm building using Xcode 26.1

I can also reproduce this issue by running `swift build` on the command line, from the root directory of libPhoneNumber-iOS:

```
$ swift --version
Apple Swift version 6.2.1 (swift-6.2.1-RELEASE)
Target: arm64-apple-macosx26.0
Build config: +assertions
$ swift build
Building for debugging...
While building module 'libPhoneNumber' imported from /Users/mathijs/Developer/libPhoneNumber-iOS/libPhoneNumberShortNumber/NBShortNumberUtil.m:11:
In file included from <module-includes>:1:
/Users/mathijs/Developer/libPhoneNumber-iOS/libPhoneNumber/libPhoneNumber.h:10:1: error: unknown type name 'FOUNDATION_EXPORT'
   10 | FOUNDATION_EXPORT double libPhoneNumber_iOSVersionNumber;
      | ^
/Users/mathijs/Developer/libPhoneNumber-iOS/libPhoneNumber/libPhoneNumber.h:13:1: error: unknown type name 'FOUNDATION_EXPORT'
   13 | FOUNDATION_EXPORT const unsigned char libPhoneNumber_iOSVersionString[];
      | ^
/Users/mathijs/Developer/libPhoneNumber-iOS/libPhoneNumberShortNumber/NBShortNumberUtil.m:11:9: fatal error: could not build module 'libPhoneNumber'
   11 | #import "NBPhoneMetaData.h"
      |  ~~~~~~~^~~~~~~~~~~~~~~~~~~
3 errors generated.
[9/27] Compiling libPhoneNumberShortNumber NBShortNumberMetadataHelper.m
$
```

I installed the Swift toolchain using the commands: `swiftly install 6.2.1 && swiftly use 6.2.1`.

The problem is resolved when I add a Foundation import statement at the top of `libPhoneNumber.h`. I've opened a pull request here: https://github.com/iziz/libPhoneNumber-iOS/pull/435

--- a/libPhoneNumber/libPhoneNumber.h
+++ b/libPhoneNumber/libPhoneNumber.h
@@ -6,6 +6,8 @@
 //  Copyright (c) 2015 ohtalk.me. All rights reserved.
 //
 
+#import <Foundation/Foundation.h>
+
 //! Project version number for libPhoneNumber-iOS.
 FOUNDATION_EXPORT double libPhoneNumber_iOSVersionNumber;
 

